Makefile

```diff
@@ -16,7 +16,7 @@ PROJECT_ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 	@grep -E '^\.PHONY: [a-zA-Z_-]+ .*?# .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = "(: |#)"}; {printf "%-30s %s\n", $$2, $$3}'
 
 .PHONY: all # Generate API, Frontend, and backend assets.
-all: preflight-checks api frontend backend-with-assets
+all: install-yarn preflight-checks api frontend backend-with-assets
```

Comment:
Looks like we already have a command for installing yarn `yarn-ensure` so we should be able to just swap out `install-yarn` with `yarn-ensure` and remove the extra make command.